iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -281,6 +301,7 @@ Inspector.prototype.update = function () {
             );
         this._updateEntityIdentifier(fact);
         this.updateCalculation(fact);
+        $('div.references').html(this._referencesHTML(fact));
```

Comment:
In general, our security team prefers if we avoid jQuery `.html`.  Is it reasonable to use `.clear().append(this._referencesHTML(fact))` or similar construct instead?

I'll admit I'm not overly familiar with jQuery, so I'm not quite following this: it looks like `Accordian.html` returns a jQuery object (not an HTML string), but it looks like jQuery `.html` only accepts HTML strings, so what does this actually do?  Does jQuery `.html` also accept jQuery objects, or is there some HTML stringifying that I'm missing?

Reply:
You're right.  I thought that _referencesHTML was returning HTML as the last line is return a.html(), but it's not - as you say, it's a jQuery object, and the docs don't suggest that this should work at all.
I've addressed this in 732b68e, and renamed the html() method to avoid confusion with the jquery .html() method.